.github/workflows/release.yml

```diff
@@ -41,13 +41,13 @@ jobs:
         run: |
           path="${PWD}/.busybox"
           mkdir -p $path
-          ln -fs /bin/busybox $path/bash
+          install -m 755 <(echo -e '#!/bin/sh\nexec busybox sh $@') $path/bash
```

Comment:
```suggestion
```
bats does not run within busybox, obviously :(

Reply:
@aslafy-z bats must be run in bash, the trick is to link the "sh" binary, since helm-git uses `#!/usr/bin/env sh`, bats will call the script with the sh binary.

I do this at helm-secrets.